roles/ovirt-affinity-groups/README.md

```diff
@@ -0,0 +1,51 @@
+oVirt affinity groups
+====================================
+
+This role manages affinity groups in oVirt.
+
+Requirements
+------------
+
+ * Ansible version 2.3 or higher
+ * Python SDK version 4 or higher
+
+Role Variables
+--------------
+
+| Name               | Default value       |                                              |
+|--------------------|---------------------|----------------------------------------------|
+| cluster            | UNDEF (Required)    |  Name of the cluster of the affinity group.  |
+| description        | UNDEF               |  Human readable description.                 |
+| host_enforcing     | false               |  If true VM cannot start on host if it does not satisfy the host_rule.|
+| host_rule          | UNDEF               |  positive: VM's in this group must run on this host. negative: VM's in this group may not run on this host |
+| hosts              | UNDEF               |  List of host names assigned to this group.  |
+| name               | UNDEF (Required)    |  Name of affinity group.                     |
+| state              | UNDEF               |  Whether group should be present or absent.  |
+| vm_enforcing       | false               |  If true, VM cannot start if it cannot satisfy the vm_rule. |
+| vm_rule            | UNDEF               |  positive: all vms in this group try to run on the same host. negative: all vms in this group try to run on separate hosts. disabled: this affinity group does not take effect. |
```

Comment:
Can you change to something like:

VM rule can have following options:
```html
<ul><li>positive - all vms in this group try to run on the same host.</li><li>negative - ll vms in this group try to run on separate hosts</li><li>disabled: this affinity group does not take effect</li></ul>
```

I hope it will be better readable for user.